packages/hardhat-test/tsconfig.types.json

```diff
@@ -1,9 +1,9 @@
 {
   "extends": "../../tsconfig.json",
   "include": ["./typechain-types"],
-  "exclude": ["node_modules"],
   "compilerOptions": {
     "esModuleInterop": true,
-    "lib": ["ES2018", "DOM"]
+    "lib": ["ES2018", "DOM"],
+    "skipLibCheck": true
```

Comment:
why this?

Reply:
There are some errors in `buffer` types clashing with our deps, and we don't really care about them in `hardhat-test`.

I'd like to move towards faster testing pipeline to boost productivity in this repo.